README.md

```diff
@@ -31,3 +32,8 @@ accordingly.
 
 What is the worst-case big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+The worst-case scenario would be if there was one path connected all the vertices and you started at the beginning and end nodes. 
+This would then mean that we have to go down every vertex in the adjacency list, taking $|V|$ time. 
```

Comment:
So the number of edges doesn't matter at all?

Reply:
Oh sorry, it would take |E| time since you’d have to iterate over every edge in that case

Reply:
So the number of nodes doesn't matter at all?

Reply:
Wel since you iterate down the adjacency list and theough each vertex’s edge, would that mean the complexity is |V| * |E|?

Reply:
No, you're not exploring all edges for all vertices.

Reply:
Oh then it would be |E| time right?

Reply:
That's what you said before. I'm getting the impression that you're just guessing at this point. Think on it for a bit.